### PR TITLE
Layout improvements

### PR DESCRIPTION
**Pull request by satbircoder**

### CODE REVIEW

File: StyleAPI.css [hunk 1]
```diff
@@ -1,60 +1,77 @@
...
-      width: 100%;
-      height:auto;
-     }
-  }
-  
-  
-  p{
-    padding: 0.5em;;
-  }
-  .content{
```
Comment (mvishnurana1) [inline StyleAPI.css:37]: Add spacing between the css classes

**Comment by mvishnurana1** [comment]
Why do we have two html and css files on the Pull Request?

**Comment by mvishnurana1** [comment]
@satbircoder can you please remove the redundant html and css file from the repo

**Comment by satbircoder** [comment]
@mvishnurana1 Files with duplicate code have been removed and recent files have been updated as well. Please refer to commit message for updated info.

File: StyleAPI.css [hunk 2]
```diff
@@ -1,60 +1,90 @@
...
 html,body{
-    height: 100%;
+  height: 100vh;
+  width: 100%;
 }
 .row{
-    height: 100%;
-    font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-    color: white;
+  display: flex;
```
Comment (mvishnurana1) [inline StyleAPI.css:11]: Test comments!

File: myAPI.html [hunk 3]
```diff
@@ -7,38 +7,62 @@
...
+                    <span>Start Course</span>
+                </button>
+                <a href="" class="link-group">
+                    <i class="fa fa-download" style="font-size:24px;">
+                    </i> 
+                    <span>
+                        Download Course
+                    </span>
+                </a>
+                <a href="" class="link-group">
```
[the remark this answers is not in the document]
Comment (satbircoder) [reply myAPI.html:42]: resolved as part of fixing another icons

**Pull request merged**